### ISSUE TITLE
Crash report 8.1: NPE in UploadService.onMediaUploaded()

### ISSUE THREAD

**Issue by aforcier** [wordpress-mobile/WordPress-Android#6529]
```
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'int org.wordpress.android.fluxc.model.PostModel.getId()' on a null object reference
       at org.wordpress.android.ui.uploads.UploadService.addUploadErrorToFailedPosts(UploadService.java:608)
       at org.wordpress.android.ui.uploads.UploadService.onMediaUploaded(UploadService.java:655)
       at java.lang.reflect.Method.invoke(Method.java)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:485)
```

Looks like we had an error for a media item that was attached to a post, but that post had been deleted by that time.

**Comment by aforcier** [comment]
> Looks like we had an error for a media item that was attached to a post, but that post had been deleted by that time.

This seems to be the case - it takes some precise timing, because we automatically emit `CANCEL_MEDIA_UPLOAD` actions for media inside deleted posts, but it's possible.

**Comment by mzorz** [comment]
Closed via #6532
